parlai/agents/image_seq2seq/modules.py

```diff
@@ -154,8 +169,11 @@ def _build_image_encoder(self):
             ]
         self.image_encoder = nn.Sequential(*image_layers)
 
-    def encode_images(
-        self, images: Union[List[object], torch.Tensor]
+    def embed_images(
```

Comment:
Hm why the name change here? It's not an embedding in the sense of a `torch.nn.Embedding`-style index lookup, for instance. And it'd be good to change the docstring if we rename the function

Reply:
my initial thought was that it sounded better during early fusion, as we call `forward_embedding` on the tokens. but yeah you're right, it's technically not similar and makes more sense as `encode`, so i'll change back